src/org/traccar/FilterHandler.java

```diff
@@ -150,16 +150,40 @@ private boolean filterLimit(Position position) {
 
     private boolean filter(Position p) {
 
+        String filterType = "";
+
         boolean result = filterInvalid(p) || filterZero(p) || filterDuplicate(p)
                 || filterFuture(p) || filterApproximate(p) || filterStatic(p)
                 || filterDistance(p);
 
+        if (filterInvalid(p)) {
+            filterType = "Invalid ";
```

Comment:
This case shouldn't be different from others. For consistency and easier modifications in future.